src/status.ts

```diff
@@ -14,11 +14,20 @@ export class RecordingStatus {
   private item: vscode.StatusBarItem;
   timeout: NodeJS.Timer;
   counting = false;
+  mainColor
```

Comment:
Please add a type declaration of `ThemeColor`.

Reply:
done